a10_octavia/controller/worker/tasks/a10_database_tasks.py

```diff
@@ -313,23 +312,6 @@ def execute(self, loadbalancer, vthunder_config):
                 loadbalancer.id)
             raise e
 
-    def revert(self, result, loadbalancer, vthunder_config, *args, **kwargs):
```

Comment:
Why was this revert removed?

Reply:
While testing I was seeing an error where when creating LB if LB goes into an error state. And when we try to delete an LB is was giving an error. This was due to this revert statement which causes vThunder entry to be removed from DB. and Delete operation does require vthunder object to make axapi calls. 
@afrin-chakure-a10 , Please confirm

Reply:
Instead of removing the revert here, let's create another bug to fix the delete operation issue

https://github.com/a10networks/a10-octavia/blob/master/a10_octavia/controller/worker/tasks/virtual_server_tasks.py#L87

We already check to see if there is a Thunder present for the virtual server delete call. So it is probably the VRRP task or VE IP task causing the issue.

Reply:
https://github.com/a10networks/a10-octavia/blob/d14308234f4956abda0a3177059d176143d73666/a10_octavia/controller/worker/tasks/a10_database_tasks.py#L316, This is a similar task here as well we don't have revert, previously revert was not there It was recently added on around 11 September 2020.

Reply:
I don't agree with that recent change and it was my bad for not catching it. We don't want to run the risk of a orphaned Thunder entry. It would be like having an orphaned Amphora.

https://github.com/openstack/octavia/blob/master/octavia/controller/worker/v2/tasks/database_tasks.py#L89

You can see upstream where they have a revert in this scenario for the Amphora. In our case, the Amphora == Thunder

Reply:
Have added the revert.